src/elements/TFhubValidator/TFhubValidator.wc.svelte

```diff
@@ -41,28 +45,32 @@
       symbol: "mnemonics",
       type: "password",
       placeholder: "For the cosmos account that will stake token.",
+      validator: syncValidateMnemonics,
       invalid: false,
     },
     {
-      label: "Stake Amount",
+      label: "Stake Amount(TFT)",
       symbol: "stakeAmount",
       type: "text",
-      placeholder: "100000000TFT",
+      placeholder: "Please note that 1 is equivalent to 10000000 TFT",
```

Comment:
this message is really confusing, 1 = 10000000 TFT while the label says that the stake amount in TFT

Reply:
yes we must remove this one from the label